drivers/ethernet/eth_e1000.c

```diff
@@ -20,6 +20,7 @@ LOG_MODULE_REGISTER(LOG_MODULE_NAME);
 
 #if defined(CONFIG_ETH_E1000_PTP_CLOCK)
 #include <zephyr/drivers/ptp_clock.h>
+#include <zephyr/irq.h>
```

Comment:
need to be outside ifdef

Reply:
thanks for spotting all those cases, my script was definitely too simple.